examples/v1beta1/cmaes-example.yaml

```diff
@@ -12,6 +12,9 @@ spec:
       - Train-accuracy
   algorithm:
     algorithmName: cmaes
+    algorithmSettings:
+      - name: "restart_strategy"
+        value: "ipop"
```

Comment:
I think we should also update the Kubeflow docs with the `restart_strategy`: https://www.kubeflow.org/docs/components/katib/experiment/#covariance-matrix-adaptation-evolution-strategy-cma-es.

Reply:
I've just opened a draft pull request at https://github.com/kubeflow/website/pull/2642 👍